### DISCUSSION TITLE
Help!  How many discussions start like this?

### DISCUSSION THREAD

**Discussion by arezee** [SecKatie/ha-wyzeapi#466] [General]
I had an issue with the wyzeapi integration in HACS as well as in Integrations...  Following some help articles I modified the manifest.json file found in wyzeapi within custom-components.  I replaced with the items starting with # bellow based on a help article I found (I then switched back to the uncommented ones) .
  "requirements": [
    "wyzeapy==0.5.16"
  ],
#  "requirements": [
#    "git+https://github.com/brg468/wyzeapy.git@update-deps#wyzeapy==0.5.19"
#  ],
After a restart of the HA server, the Wyze Integration is no longer visible in HA Integrations. I cannot remove wyzeapi in HACS and, 
 adding the repository "https://github.com/JoshuaMulliken/ha-wyzeapi.git" in HACS reports that the repository is already in the store ( I do not know how to remove it but attempted editing .storage/hacs.repositories to remove the wyze integration).  I attempted to rename the wyzeapi folder in custom-components (xxxxxxxwizeapi) and even downloading the zip file from the repository, extracting and copying to a new wyzeapi folder then installing the integration in HA getting an error "Config flow could not be loaded: 500 Internal Server Error Server got itself in trouble"
I'd like to continue to use the HACS wyzeapi and the integration if possible.  recommendations to clean up and get it working?
Thanks.
Raphael

**Comment by arezee** [comment]
This is the uncommented part that was in he original post
 
 "requirements": [
    "wyzeapy==0.5.16"
  ],

This is the commented part in the original post that I tested
 
"requirements": [
   "git+https://github.com/brg468/wyzeapy.git@update-deps#wyzeapy==0.5.19"
 ],

Thanks

**Comment by SecKatie** [comment]
The latest update fixes this bug. It was caused by Home Assistant blocking one of our dependencies.

**Comment by arezee** [comment]
Thank you.  Applied and working
R.



Sent from my T-Mobile 5G Device



-------- Original message --------
From: Joshua Mulliken ***@***.***>
Date: 3/9/23 2:21 PM (GMT-05:00)
To: JoshuaMulliken/ha-wyzeapi ***@***.***>
Cc: Raphael Zagha ***@***.***>, Author ***@***.***>
Subject: Re: [JoshuaMulliken/ha-wyzeapi] Help! How many discussions start like this? (Discussion #466)


The latest update fixes this bug. It was caused by Home Assistant blocking one of our dependencies.
